app/front/scripts/services/history.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
@akariv @borysyuk is working on this right now in https://github.com/openspending/os-viewer/issues/61